### PR TITLE
Refactored dropdown item events

### LINKED ISSUE #3382: Correct onClick handling in Dropdowns
Many dropdown-items do not use their own onClick property. Instead the content of the item is wrapped in an div-container that has the onClick property. `tacing-actions-view` is a good example for this.

Hint:
Be careful: Moving the onClick into the item can create side effects, like the event object passed to the handle-function does no longer have a traget that can blurred.

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
This PR moves the onClick handling of dropdown menu items out of the inner containers like div and span directly into the items. Thus making the code more clear and increasing the area that triggers the onClick event. These changes are also needed for issue  #3319 because for the fiddling with the layout of dropdown menus a consistency is needed.

### URL of deployed dev instance (used for testing):
- [https://refactordropdownitemevents.webknossos.xyz](https://refactordropdownitemevents.webknossos.xyz)

### Steps to test:
- check all dropdown items that where changed to trigger their onClick event correctly. 
- Also there should be no visual changes.

- check whether I found all dropdown menus and checked them.
I checked: 
    - tracing_actions_view
    - dataset_action_view (none)
    - view_modes_view (none)
    - comment_tab_view (none)
    - tree_hierarchy_view (none)
    - tree_tab_view
### comment
Please note that I removed `event.target.blur();` because the event that is passed to the called function no longer has the property target. And I am pretty sure that antd already handles the blur.
### Issues:
- fixes #3382 

------
- ~~[ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

**Comment by philippotto** [comment]
Nice work!

> Please note that I removed event.target.blur(); because the event that is passed to the called function no longer has the property target. And I am pretty sure that antd already handles the blur.

Antd doesn't do blur, since this is not standard behavior. Usually, clicking on an element (e.g., a button) will give that element focus. However, in wk core, we usually want the tracing viewports to have focus. That's why, we blur the clicked element, so that the focus is returned to the tracing view.

**Comment by philippotto** [comment]
Nice work!

> Please note that I removed event.target.blur(); because the event that is passed to the called function no longer has the property target. And I am pretty sure that antd already handles the blur.

Antd doesn't do blur, since this is not standard behavior. Usually, clicking on an element (e.g., a button) will give that element focus. However, in wk core, we usually want the tracing viewports to have focus. That's why, we blur the clicked element, so that the focus is returned to the tracing view.

File: app/assets/javascripts/oxalis/view/right-menu/trees_tab_view.js [hunk 1]
```diff
@@ -187,23 +187,23 @@ class TreesTabView extends React.PureComponent<Props, State> {
   getActionsDropdown() {
     return (
       <Menu>
-        <Menu.Item key="shuffleTreeColor">
-          <div onClick={this.shuffleTreeColor} title="Change Tree Color">
+        <Menu.Item key="shuffleTreeColor" onClick={this.shuffleTreeColor}>
+          <div title="Change Tree Color">
```
Comment (daniel-wer) [inline app/assets/javascripts/oxalis/view/right-menu/trees_tab_view.js]: Did you try to move the title to the Menu.Item as well? This way we could get rid of the wrapper divs for this and the next Menu.Items :)

**Comment by daniel-wer** [comment]
@MichaelBuessemeyer I did a search in the code and found one remaining file where this could be improved, the `task_annotation_view` (`<Item ...>`). If you find the time you could fix the issue there as well :)

**Comment by daniel-wer** [comment]
And btw, I no longer had the problem that clicking near the border would not activate the dropdown item (due to the padding). I think moving the onClick fixed that already, so there is no need for #3319, do you agree? :)

**Pull request merged**